modules/layers/src/path-layer/path-layer.js

```diff
@@ -125,13 +126,15 @@ export default class PathLayer extends Layer {
     const geometryChanged =
       changeFlags.dataChanged ||
       props.fp64 !== oldProps.fp64 ||
+      props.is3D !== oldProps.is3D ||
```

Comment:
Nit: I never like the need to give special consideration to props here, it feels fragile and easy to get wrong and maintain across layers.

I'd be fine if we specified the `is3D` prop as a prop that is consulted when data changes, not a prop that triggers a change in itself.

Any use case for this trigger would seem artificial (i.e. this if statement supports the case when data hasn't changed, we just change interpretation from 2 to 3 dimensions).